attending/library.py

```diff
@@ -74,3 +75,81 @@ def __contains__(self, module):
     @_requires_valid_module
     def __getitem__(self, module):
         return self.docs[module.__name__][module]
+
+
+def fetch(module, version=None)
+    """
+    A convenience top-level function for fetching docs
+
+    Parameters
+    ----------
+    module : string or module
+        The module whose docs we want to pull up
+    version: string, optional
+        The version for `module`, will fall-back to latest, if not specified.
+    """
+    from types import ModuleType
+    if isinstance(module, ModuleType):
+        return fetch_via_module(module, version)
+    else:
+        return fetch_via_name(module, version)
+
+
+def fetch_via_module(module, version=None):
+    """
+    Fetch the docs for a given imported module
+
+    Parameters
+    ----------
+    module : module
+        The module whose docs we want to pull up
+    version: string, optional
+        The version for `module`, will fall-back to current, if not specified.
+    """
+    lib = Library()
+    mod_name = module.__name__
+
+    if version is None:
+        # First, let's try
+        version = module.__dict__.get('__version__')
+
+    if version is None:
+        # Couldn't get __version__ from module
+        # We can either:
+        # 1. Assume it's the latest from PyPI
```

Comment:
i haven't even thought about conda until now. just a thought for the future.

Reply:
the reason for PyPI here is because of the JSON API there that has a documentation URL.